setup/stock_move_change_source_location/odoo/addons/stock_move_change_source_location

```diff
@@ -0,0 +1 @@
+../../../../stock_move_change_source_location
```

Comment:
For next time, run pre-commit before doing the migration commit

Reply:
My philosophy is that if the pre-commit commit will not have any changes (except the setup folder thing) then let's go directly to the migration commit to avoid adding extra commits hehe